metabase/metabase
# Hide "Add another series" button when there are no series to add

Fixes https://github.com/metabase/metabase/issues/11249
Reproduces https://github.com/metabase/metabase/issues/11249

How to test:
- Create a question based on Orders, grouped by Created at (month), sum by Count and Avg (Total)
- Go to Settings -> Data -> Remove series for Avg (Total)
- Click `Add another series...`, it should add a series for Avg (Total) automatically
- `Add another series...` should no longer be visible

<img width="1159" alt="Screenshot 2022-04-29 at 19 57 31" src="https://user-images.githubusercontent.com/8542534/165989669-c9bf68b9-fca4-42df-b5b0-2792dd744e74.png">
